src/lib/chips/chip-list.ts

```diff
@@ -274,12 +274,17 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
   @Input('aria-orientation') ariaOrientation: 'horizontal' | 'vertical' = 'horizontal';
 
   /**
-   * Whether or not this chip is selectable. When a chip is not selectable,
+   * Whether or not this chips are selectable. When a chip is not selectable,
```

Comment:
I think the comment still refers to an individual chip, but this is in the chip list.